src/components/LogIn/LogIn.js

```diff
@@ -45,7 +46,8 @@ export default function LogIn() {
                     password: password,
                 })
                 .then((response) => {
-                    console.log(response.data);
+                    window.localStorage.setItem('token', response.data.token);
+                    history.push('/cart');
```

Comment:
can you also store user object to local storage?

Reply:
ok. let me add it